projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap.sh

```diff
@@ -18,6 +18,24 @@ then
     VIP=$2
 fi
 
+# static ip configuration needs 5 parameters for cp nodes
+if [ $# -eq 5 ]
+then
+    KUBE_VIP_IMAGE=$1
+    VIP=$2
+    STATIC_IP=$3
+    GATEWAY=$4
+    CIDR=$5
```

Comment:
Could we just receive the argument in the same order irregardless of how many args?

```suggestion
    STATIC_IP=$1
    GATEWAY=$2
    CIDR=$3
    KUBE_VIP_IMAGE=$4
    VIP=$5
```